ci-operator/step-registry/optional-operators/configure-proxy-registry/optional-operators-configure-proxy-registry-ref.yaml

```diff
@@ -7,8 +7,8 @@ ref:
       cpu: 300m
       memory: 300Mi
   credentials:
-  - namespace: brew-registry-pullsecret # this entry injects the custom credential
-    name: registry-pullsecret
+  - namespace: test-credentials # this entry injects the custom credential
+    name: brew-registry-pullsecret
```

Comment:
We need to make sure the secret is already present there before we make this change, I'd like to avoid breaking any in-flight job.

Reply:
https://github.com/openshift/release/pull/16495

Reply:
Where do we check if there is any job in flight?